epics/cache.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
I've put all the epic tests into `middleware/epic.test.js` in order to simulate `dispatching` the actions, but I've just realized that's not necessary since Epics are pure function that can be tested independently of the middleware... is it alright with you if I file a followup with convert the `epic.test.js` unit tests to epic-specific test scripts?